R/cv_varsel.R

```diff
@@ -644,18 +642,15 @@ loo_varsel <- function(refmodel, method, nterms_max, ndraws,
       log_ratios = -loglik_forPSIS
     )
     # The k-values are h-specific (expectation-specific) here (see Vehtari et
-    # al., 2022, <https://doi.org/10.48550/arXiv.1507.02646>, beginning of
+    # al., 2024, <https://doi.org/10.48550/arXiv.1507.02646>, beginning of
```

Comment:
Yey! :) Just noting that the DOI needs to be updated and perhaps this reference occurs at other places as well where it needs to be updated, too.

Reply:
fixed